src/SkySensorsAPI/Middlewares/ExceptionHandlerMiddleware.cs

```diff
@@ -11,9 +12,10 @@ public async Task Invoke(HttpContext context)
 			await nextMiddleware.Invoke(context);
 		}
 		catch (Exception ex) when (
-			ex is FormatException || 
-			ex is JsonException || 
-			ex is ArgumentException)
+			ex is FormatException ||
+			ex is JsonException ||
+			ex is ArgumentException ||
+			ex is PostgresException)
```

Comment:
Har du en mulighed for at tjekke for en bestemt fejl besked lige som nedenunder da PostgresException er meget abstrakt?
`catch (InvalidOperationException ex) when (ex.Message == "Sequence contains no elements")`

Reply:
Ja